js-exercises/alphabetic-shift/alphabeticShift.test.js

```diff
@@ -0,0 +1,20 @@
+import { alphabeticShift } from './alphabeticShift';
+
+describe('Alphabetic Shift Test Cases', () => {
+  it('returns a shifted string', () => {
+    expect(alphabeticShift('Pseudopseudohypoparathyroidism'))
+      .toEqual('Qtfvepqtfvepizqpqbsbuizspjejtn');
+
+    expect(alphabeticShift('Floccinaucinihilipilification'))
+      .toEqual('Gmpddjobvdjojijmjqjmjgjdbujpo');
+
+    expect(alphabeticShift('Floccinaucinihilipilification'))
+      .toEqual('Gmpddjobvdjojijmjqjmjgjdbujpo');
+
+    expect(alphabeticShift('Antidisestablishmentarianism'))
+      .toEqual('Boujejtftubcmjtinfoubsjbojtn');
+
+    expect(alphabeticShift('supercalifragilisticexpialidocious'))
```

Comment:
Just a suggestion: Do write at least 1 of your own test case. It builds that habit of writing tests.

Reply:
Okay  :)